specification/loadtestservice/LoadTestService/main.tsp

```diff
@@ -0,0 +1,47 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@typespec/versioning";
+import "./routes.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+using Azure.Core;
+
+@versioned(APIVersions)
+@useAuth(Oauth2)
+@service({
+  title: "Azure Load Testing",
+})
+@server(
+  "https://{endpoint}",
```

Comment:
The endpoint folks get from the portal doesnt include the scheme?
```suggestion
  "{endpoint}",
```

Reply:
Our resource endpoint from portal doesn't include scheme, therefore added https://